game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapRouteDrawer.java

```diff
@@ -177,13 +177,10 @@ private void drawDirectPath(final Graphics2D graphics, final Point2D start, fina
    */
   protected double[] newParameterizedIndex(final Point2D[] points) {
     final double[] index = new double[points.length];
-    if (index.length > 0) {
-      index[0] = 0;
-    }
```

Comment:
Looks like I was coding way too much C++ when I was originally writing this code.
In java obviously the array is initialized with 0.